src/pages/Cart/Cart.scss

```
@@ -1 +1,98 @@
-//q
+@import '../../styles/mixin';
+
+button:hover {
+  cursor: pointer;
+}
+
+.cart {
+  padding: 40px;
+  @include textStyle(20px, 500);
+
+  .title {
+    margin-bottom: 50px;
+    font-size: 30px;
+  }
+
+  .cart-container {
+    display: flex;
+    margin-top: 10px;
+  }
+
+  .cart-main {
+    width: 60%;
+    margin-right: 40px;
+
+    .cart-box {
+      display: flex;
+      padding: 25px 0 20px 0;
+      border-bottom: 1px solid #dfdfdf;
+
+      .cart-img {
+        width: 35%;
+        margin-right: 10px;
+      }
+
+      .btn {
+        padding: 0;
+        border: none;
+        background-color: white;
+        font-size: 20px;
+      }
+
+      .cart-content {
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+        padding: 10px;
+      }
+
+      .content {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        justify-content: space-between;
+      }
+
+      .img {
+        width: 100%;
+      }
+
+      .total {
+        display: flex;
+        flex-direction: column-reverse;
+      }
+    }
+  }
+
+  .cart-aside {
+    width: 40%;
+    padding-top: 25px;
+
+    .price {
+      padding: 10px 0 10px 0;
```

```suggestion
      padding: 10px 0;
```
동일한 스타일이라면 이렇게 적용할 수 있습니다!